src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.add = (x) => {
+    calculator.result += x;
+
+    return calculator;
+  };
```

Comment:
```suggestion
  const calculator = {
   result: 0,

   add: (x) => {
    this.result += x;

    return this;
  };
```
компактніше було б одразу в об'єкт прописати всі поля і методи, використовуючи this

Reply:
Ок, зрозумів)) зараз зробимо!